packages/grafana-ui/src/components/PageLayout/PageToolbar.tsx

```diff
@@ -50,7 +50,7 @@ export const PageToolbar = React.memo(
     /**
      * .page-toolbar css class is used for some legacy css view modes (TV/Kiosk) and
      * media queries for mobile view when toolbar needs left padding to make room
-     * for mobile menu icon. This logic hopefylly can be changed when we move to a full react
+     * for mobile menu icon. This logic hopefully can be changed when we move to a full react
```

Comment:
🙌

Reply:
the spell checker kept screaming at me if I didn't change it :sweat_smile: